addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -924,6 +949,8 @@ const actions = {
             }));
             dispatch('_loadNewMessagesOnThread', thread.localId);
         }
+        dispatch('resetComposer', composerLocalId);
+        dispatch('unlinkAttachmentsFromComposer', composerLocalId);
```

Comment:
same, and move it inside `_resetComposer` since only used there (and it makes sense to always unlink attachments from composer from a reset)

Reply:
Done